lib/js/_create/_build-new-package.js

```diff
@@ -36,6 +36,15 @@ function generateFolders(answers) {
 					if (err) {
 						console.error(err);
 					} else {
+						if (value === 'scss') {
+							['10-settings', '20-functions', '30-mixins', '40-base', '50-components', '60-utilities'].forEach(subfolder => {
+								fs.mkdir(`${filePath}/${answers.pkgname}/${value}/${subfolder}`, err => {
+									if (err) {
```

Comment:
can you write tests for the new functionality
https://github.com/springernature/frontend-package-manager/blob/master/__tests__/unit/_create/build-new-package.js

Reply:
Also, I have tried to keep this package free from hard-coding options like this. If we were ever to change how we write stuff, then someone would have to know to dig into here in order to re-write this. Also, this is potentially useful for other projects that do not follow the same structure (other domains, possibly). So we need to think about how to make this configurable.

Reply:
my feeling is this should be part of the config somehow
e.g. https://github.com/springernature/frontend-global-toolkit/blob/master/package-manager.json

Reply:
I've tried to do it through the config. It seems to work.
Could we pair on writing a test? I'm struggling with it 😞

Reply:
Sure, I'm not in the office today, but we can sit down and look at it tomorrow